src/main/webapp/app/overview/course-exercises/course-exercise-row.component.ts

```diff
@@ -136,8 +136,16 @@ export class CourseExerciseRowComponent implements OnInit, OnDestroy {
             }
         } else if ((exercise.type === ExerciseType.MODELING || exercise.type === ExerciseType.TEXT) && this.hasParticipations(exercise)) {
```

Comment:
Can you also add here File Upload Exercise? This is a newly added type of exercise, but it should be treated the same way as text and modeling exercises.

Reply:
Thanks again for your review! Okay, if file upload exercises really work the same way, we can apply the logic even in this PR :-) I adjusted my implementation at all places you mentioned and at two more. I tested it locally for file upload exercises and critical use cases and it seems to work fine. I'll test it more extensively on the test server but since you are more familiar with the desired behavior of file upload exercises, it would be grate if you would test again briefly :-)